version.py

```diff
@@ -0,0 +1,21 @@
+﻿import discord
+
+class IVersionProvider:
+	def get_versions_and_embeds(self):
+		pass
+
+class VersionProviderImpl(IVersionProvider):
+	def __init__(self):
+		self.map = {};
+
+		# Version 2.0.0
+		v2 = discord.Embed(title="New features in v 2.0.0!")
+		
+		v2.add_field(inline=False, name="Vote Timer", value="You can now use `!votetimer 5m30s` to start a timer for the requested time. Villagers will be warned as the timer gets close, and when it expires, villagers will be pulled to the Town Square channel for a vote as if `!vote` was run.\nYou'll need to run `!setChatChannel` to see the warning messages.")
```

Comment:
I think you can leave off the `!setChatChannel` part - if they try to run the vote timer, it will tell them that they need to do that.